core/helper-tools/src/main/java/esa/mo/helpertools/helpers/HelperAttributes.java

```diff
@@ -75,36 +75,36 @@
   public static Double attribute2double(Attribute in) {
 
     if (in instanceof Union) {
-      if (((Union) in).getTypeShortForm().equals(Union.BOOLEAN_TYPE_SHORT_FORM)) { // 2
+      if (in.getTypeShortForm().equals(Union.BOOLEAN_TYPE_SHORT_FORM)) { // 2
```

Comment:
Hey @arturobernalg, are you sure about those automatic refactor changes?

Looking at this first change it seems to break stuff: 
- [Attribute.java](https://github.com/esa/mo-services-java/blob/master/apis/api-mal/src/main/java/org/ccsds/moims/mo/mal/structures/Attribute.java) doesn't contain any `getTypeShortForm()` method.

I guess the wrong automatic change could come from the `@Override` annotation on `getTypeShortForm()` method in [Union.java](https://github.com/esa/mo-services-java/blob/master/apis/api-mal/src/main/java/org/ccsds/moims/mo/mal/structures/Union.java) that is incorrect since it doesn't implement or override anything from `Attribute` class.